model/things/src/test/java/org/eclipse/ditto/model/things/ImmutableThingTest.java

```diff
@@ -1115,10 +1124,22 @@ public void createThingWithInvalidCharactersInId() {
                                 FEATURES,
                                 LIFECYCLE,
                                 REVISION,
-                                MODIFIED)
+                                MODIFIED,
+                                METADATA)
                 )
         );
 
     }
 
+    @Test
+    public void serializeMetadata() {
+        final JsonObject json = KNOWN_THING_V2.toJson(JsonSchemaVersion.V_2, field -> true);
```

Comment:
I would prefer the following implementation of that method:

    final JsonObject json = KNOWN_THING_V2.toJson(JsonSchemaVersion.V_2, field -> true);

    final Optional<JsonObject> metadataOptional = json.getValue(Thing.JsonFields.METADATA);

    assertThat(metadataOptional).contains(JsonObject.newBuilder().set("_issuedAt", JsonValue.of(0)).build());